src/staticresources/DefaultGiftEntryTemplate.resource

```diff
@@ -1 +1 @@
83bf4d641a1","label":"Address: Mailing Zip/Postal Code","objectApiName":"Address__c","required":false,"sectionId":"71dd7e32-3408-4d49-ea5c-d3dbd6374ba1"},{"customLabel":"Contact Country","dataImportFieldMappingDevNames":["Home_Country_62bf1425e"],"dataType":"STRING","elementType":"field","fieldApiName":"MailingCountry__c","id":"85aa4958-2179-ec8b-7fc2-b9b8136b070a","label":"Address: Mailing Country","objectApiName":"Address__c","required":false,"sectionId":"71dd7e32-3408-4d49-ea5c-d3dbd6374ba1"}],"id":"71dd7e32-3408-4d49-ea5c-d3dbd6374ba1","label":"Add or Edit Contact"},{"defaultDisplayMode":"expanded","displayRule":"displayRule","displayType":"accordion","elements":[{"customLabel":"Donation Date","dataImportFieldMappingDevNames":["Donation_Date_1747b8d1a"],"dataType":"DATE","elementType":"field","fieldApiName":"CloseDate","id":"6aee78f5-1b92-3d4f-a783-8c14990de773","label":"Opportunity: Close Date","objectApiName":"Opportunity","required":true,"sectionId":"349da17f-20fc-de89-ad04-3bcfea525573"},{"customLabel":"Donation Amount","dataImportFieldMappingDevNames":["Donation_Amount_9ccc6d19e"],"dataType":"CURRENCY","elementType":"field","fieldApiName":"Amount","id":"71e9efde-7be3-974a-5503-ceec76768afe","label":"Opportunity: Amount","objectApiName":"Opportunity","required":true,"sectionId":"349da17f-20fc-de89-ad04-3bcfea525573"},{"customLabel":"Primary Campaign Source","dataImportFieldMappingDevNames":["Donation_Campaign_Source_2f299ad43"],"dataType":"REFERENCE","elementType":"field","fieldApiName":"CampaignId","id":"0b35ad54-3db3-32ed-0fa1-4e6df8de38d9","label":"Opportunity: Campaign ID","objectApiName":"Opportunity","required":false,"sectionId":"349da17f-20fc-de89-ad04-3bcfea525573"},{"customLabel":"Payment Method","dataImportFieldMappingDevNames":["Payment_Method_ddfc29c2a"],"dataType":"PICKLIST","elementType":"field","fieldApiName":"npe01__Payment_Method__c","id":"f058916d-630f-f033-8bc1-777d2dc27ac3","label":"Payment: Payment Method","objectApiName":"npe01__OppPayment__c","required":false,"sectionId":"349da17f-20fc-de89-ad04-3bcfea525573"},{"customLabel":"Check/Reference Number","dataImportFieldMappingDevNames":["Payment_Check_Reference_Number_e32ddab6a"],"dataType":"STRING","elementType":"field","fieldApiName":"npe01__Check_Reference_Number__c","id":"921b5649-5e83-d7d1-eca6-4ddffd7e66c6","label":"Payment: Check/Reference Number","objectApiName":"npe01__OppPayment__c","required":false,"sectionId":"349da17f-20fc-de89-ad04-3bcfea525573"},{"id":"bd284b44-7009-87d3-238a-73bc9eb9dd33","componentName":"geFormWidgetAllocation","label":"Allocations","customLabel":"Allocations","required":false,"sectionId":"349da17f-20fc-de89-ad04-3bcfea525573","elementType":"widget","dataImportObjectMappingDevName":"GAU_Allocation_1_8f","dataImportFieldMappingDevNames":["GAU_Allocation_1_GAU_3e","GAU_Allocation_1_Amount_70","GAU_Allocation_1_Percent_61"]}],"id":"349da17f-20fc-de89-ad04-3bcfea525573","label":"Donation Information"}]},"name":"Default Gift Entry Template"}
```

Comment:
Should we include the namespace here in the JSON and use the newer `alignSchemaNSWithEnvironment` method instead of `StrTokenNSPrefix` ?

Reply:
Longer term, yes - but I think for now I want to move ahead with this version to unblock us if that's ok